public_html/assets/css/styles.css

```diff
@@ -336,16 +318,10 @@ section#team .hover-group-inner {
 }
 
 /* all stuff that helps with this damned cursor easter egg. delete this to get rid of it. */
-section#team a.item[data-url="carolina-pistone"],
-section#team a.item[data-url="lauren-golik"] {overflow:visible;}
-section#team a.item[data-url="carolina-pistone"]::before,
-section#team a.item[data-url="carolina-pistone"]::after,
-section#team a.item[data-url="lauren-golik"]::before,
-section#team a.item[data-url="lauren-golik"]::after {cursor:url("../img/TEAMMATES/teammate-nathan-sticker.gif"), auto; content:""; display:block; position:absolute; left:-5px; top:0; width:5px; bottom:0; z-index:1;}
-section#team a.item[data-url="carolina-pistone"]::after,
-section#team a.item[data-url="lauren-golik"]::after {right:-5px; left:auto;}
-
-
+section#team a.item {overflow:visible;}
+section#team a.item::before,
+section#team a.item::after {cursor:url("../img/TEAMMATES/teammate-nathan-sticker.gif"), auto; content:""; display:block; position:absolute; left:-5px; top:0; width:5px; bottom:0; z-index:1;}
+section#team a.item::after {right:-5px; left:auto;}
```

Comment:
when you're a sticker, you need to show your face more. this puts my face on the edge of everyone's photo :)